### PR TITLE
HBW-137 Add duration of requests logging

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
```[HOMS] I, [2019-04-24T17:51:20.821549 #25287]  INFO -- : [DURATION] 569.685ms, GET /widget/buttons?entity_code=ORD-5&entity_type=pizza_order&entity_class=order```

File: hbw/app/controllers/hbw/base_controller.rb [hunk 1]
```diff
@@ -2,11 +2,23 @@ module HBW
...
     end
+
+    private
+
+    def start
+      @start = Time.now.to_f
+    end
+
+    def log
+      logger.tagged('DURATION') { logger.info { "#{(Time.now.to_f - @start) * 1000.0}ms, #{request.method} #{request.original_fullpath}" } }
```
Comment (sugarfree1) [inline hbw/app/controllers/hbw/base_controller.rb]: Create a separate logging module. Include it and use here.

**Comment by mxsmirnoff** [comment]
debug - 
 ```[HBW] D, [2019-04-26T15:37:09.089320 #44542] DEBUG -- : [DURATION] 371.3099956512451ms, GET /widget/buttons?entity_code=ORD-4&entity_type=pizza_order&entity_class=order```
info - 
```
 [HBW] I, [2019-04-26T15:40:45.230473 #46747]  INFO -- : [DURATION] 127.94804573059082ms, GET /widget/users/check
```

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**